### PR TITLE
add: information about the defined relationships

### PR DESCRIPTION
**Pull request by RipulHandoo**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/concepts/logical/relationshipsInfo.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: enhanced
+title: Relationships Information
+permalink: concepts/logical/relationshipsInfo
```
Comment (leecalcote) [inline docs/pages/concepts/logical/relationshipsInfo.md:4]: From the description of this page, it seems that this page is most appropriately located under Contributing.

There can be two versions of this page that refer to the same relationship data.

@iArchitSharma @vishalvivekm

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/concepts/logical/relationshipsInfo.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/concepts/logical/relationshipsInfo.md:4]: @RipulHandoo, please relocate and rename the page, too, using hyphenation where necessary.

**Comment by iArchitSharma** [comment]
Hi @RipulHandoo, are you stuck? How can I help you?

**Comment by RipulHandoo** [comment]
@iArchitSharma We need to relocate the relationship page from user-facing docs to contributor-facing docs. @pandeyshubham03 has an open PR where he is working on providing information on how different components interconnect with each other (basically listing all the relationships we have defined so far).

We can proceed to merge this PR with the relationships.md file.

File: docs/pages/concepts/logical/relationshipsInfo.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <th>Cavets</th>
+    </tr>
+    <tr>
+        <td>Pod</td>
+        <td>Deployment, StatefulSet, DaemonSet, ReplicaSet</td>
+        <td>Hierarchical</td>
+        <td>Inventory</td>
+        <td>heirarchical_inventory_relationship</td>
+        <td>While defining a relationship between a pod and a deployment, any configurations added to the pod are automatically incorporated into the deployment manifest. Consequently, users don't need to separately specify configurations for the deployment.</td>
+        <td>After the relationship is established. Unfortunately, there's no system to remove the extra pod configuration automatically in this situation. This can result in duplication, where both the pod and deployment have similar configurations. It's important to be aware of this possibility and manage configurations carefully to avoid unexpected issues during deployment.</td>
```
Comment (leecalcote) [inline docs/pages/concepts/logical/relationshipsInfo.md:32]: This is an important caveat to include an alert. e.g. `{% include alert.html type="warning" content="...`

File: docs/pages/concepts/logical/relationships.md [hunk 3]
```diff
@@ -20,15 +20,16 @@ Meshery supports a variety of relationships between components. Relationships ar
...
-1. Edge
-   1. Network
-   1. Firewall
-   1. Binding
-      1. Mount
-      1. Permission
-1. Heirarchical
-   1. Inventory
-   1. Parent
+1. `Edge` : Edge relationships indicate the possibility of traffic flow between two components. Edge relationship can be divided into 3 sub types.
```
Comment (MUzairS15) [inline docs/pages/concepts/logical/relationships.md:23]: Look for the description for these in contributing docs and expand upon them.
The current descriptions for Network, Firewall... is not enough.

**Review by MUzairS15 — COMMENTED** [review]
@vishalvivekm @iArchitSharma What will be the location and format we will be having for automatic publishing of relationship docs?

**Comment by iArchitSharma** [comment]
@MUzairS15  last time we decided to have the collection for the relationships in `.md` files under a new directory `_relationship`
The md file will contain the yml data from relationship spreadsheet

Right, @vishalvivekm? What do you say?

**Comment by leecalcote** [comment]
Gents, which way are we headed on this? @vishalvivekm?

File: docs/pages/concepts/logical/relationshipsInfo.md (hunk 2 above)
Comment (MUzairS15) [reply docs/pages/concepts/logical/relationshipsInfo.md:32]: @RipulHandoo

Comment (MUzairS15) [reply docs/pages/concepts/logical/relationshipsInfo.md:32]: @vishalvivekm @RipulHandoo This is an important caveat causing this PR to be on hold.
https://github.com/meshery/meshery/pull/10332#pullrequestreview-1899545539

**Comment by MUzairS15** [comment]
Closing as addressed in #10332

**Pull request closed without merging**